examples/example-2.xml

```diff
@@ -28,7 +28,7 @@
   </metadata>
 
   <results>
-    <issue cwe="401" test-id="refcount-too-high">
+    <issue cwe="401,200" test-id="refcount-too-high">
 
```

Comment:
Thanks for posting this.

Could you please add a fresh example e.g. taken from the flawfinder example you posted a screenshot of?

I'm not keen on having comma-separated values in the XML; I think it would be cleaner to introduce elements for these IDs, so that client code doesn't have to parse the attributes.  Instead we should add a new zero-or-more child element to the XML schema.

We might want to support other categorization schemes - the readme.rst talks about:
> potentially with other IDs and URLs, e.g. the ID "SIG30-C" with
> URL https://www.securecoding.cert.org/confluence/display/seccode/SIG30-C.+Call+only+asynchronous-safe+functions+within+signal+handlers
so maybe we want to generalize things accordingly.